Date: 2013-04-16
Title: MDC Interview Week Spring 2013, Max Delbrueck Centrum, Berlin Buch
Summary: Slides for the interview week at MDC
Abstract: My presentation for the lab meeting April 15 - 18, 2013
Template: deck
Tags: research, slides, hd dna-fish, phd, mdc, deck
Affiliation: Master in Molecular Biosciences, 	
Place: University of Heidelberg
Image_top: hdfish.png
Category: showcase
Class: entry2

<div class="slide" id="1"><h2>Chromatin topology plays an important role in gene regulation</h2>
<p><img alt="slide1" src="/static/images/slides/chromatin-organization/slide1_2.png" /></p>
</div><div class="slide" id="2"><h2>Distance between chromatin regions are measured from 3D positions of fluorescent spots</h2>
<p><img alt="slide2" src="/static/images/slides/chromatin-organization/slide2.png" /></p>
</div><div class="slide" id="3"><h2>Spots are identified for singles cells</h2>
<p><img alt="spotterUI2" src="/static/images/slides/chromatin-organization/spotterUI.png" /></p>
</div><div class="slide" id="4"><h2>Precise localization of the spots is estimated and corrected for chromatic aberrations</h2>
<p><img alt="gaussfit" src="/static/images/slides/chromatin-organization/slide4.png" /></p>
</div><div class="slide" id="5"><h2>Distances within single chromosomes are measured</h2>
<p><img alt="distances" src="/static/images/slides/chromatin-organization/slide5.png" /></p>
</div><div class="slide" id="6"><h2>Our results are in agreement with previous observations</h2>
<p><img alt="4 loci counts" src="/static/images/slides/chromatin-organization/slide6.png" /></p>
<pre style="text-align:right;">Mateos-Langerak et al. 2009 PNAS</pre>

<h3>Outlook</h3>
<h4>Combinatorial labeling provides identifiers for unique regions</h4>
<p><img alt="distances" src="/static/images/slides/chromatin-organization/comb-label.png" /></p>
</div><div class="slide" id="7"><h2>Concluding remarks</h2>
<ul>
<li>
<p>A major question in understanding spatial genome organization is how patterns of chromosome or loci organization are established and maintained</p>
</li>
<li>
<p>Better resolution and perhaps multiplexing for the precise estimation of the localization of chromosome regions can be acheived by using our methodology </p>
</li>
<li>
<p>These measurements will provide insight in understanding the formation of patterns in chromatin organization</p>
</li>
</ul>
</div><div class="slide" id="8"><h2>Aknowledgements</h2>
<h3>Group van Oudenaarden</h3>
<h4>at MIT:</h4>
<ul>
<li>Magda Bienko</li>
<li>Stefan Semrau</li>
</ul>
<h4>at the Hubrecht Institute:</h4>
<ul>
<li>Alexander van Oudenaarden</li>
</ul>
</div><div class="slide" id="9"><h2>try come code:</h2>
<div class="codehilite"><pre>begin<span class="nb">{</span>equation<span class="nb">}</span>
   E = mc<span class="nb">^</span>2
   e<span class="nb">^{</span><span class="k">\pi</span> i<span class="nb">}</span>+1 = 0 <span class="k">\frac</span><span class="nb">{</span>x<span class="nb">}{</span>y<span class="nb">}</span>
end<span class="nb">{</span>equation<span class="nb">}</span>
</pre></div>


<h3>is translated into:</h3>
<p>\begin{equation}
E = mc^2 \
e^{\pi i}+1 = 0 \frac{x}{y}
\end{equation}</p>
<p></div><div class="slide" id="30" style="background-color:#000000;"><h1 style="color:#fafafa;">Thank you!</h1></p>
</div>
