Title: Research Interests and Experience
Date: 2013-11-05
Category: personal
Tags: EMBL

* **Research Experience.** Write the title and a description of your Diploma (thesis) research, or equivalent. Indicate the significance, the aims and your contributions and, in all cases, state the number of months of research involved. If you carried out other research prior to your Diploma, or since then, please describe it briefly in a separate paragraph. List publications, if any (minimum 500 and maximum 2000 characters).

* **Scientific Interests.** Please write a personal statement about your scientific interests.To make your point, you may mention specific EMBL group leaders who you would be interested in working with (these may include groups that are outside of the units you have chosen on the 'programme preference' page). Please explain clearly why you would like to work in these areas of research. Please note that this part of the application serves two purposes: It allows us to find out what your scientific interests are, and it gives you the possibility to highlight group leaders whose work you are particularly interested in, including some who might not be part of the two units you selected.(minimum 500 and maximum 2000 characters)

---

## Research Experience

My research experience hasn't been limited to a single project; luckily, throughout my carrer --both Bachelor and now Master-- I have had the opportunity to occupy myself with different topics and interact with various research groups. First at the National Laboratory of Genomics for Biodiversity in Mexico at the Genetics Systems Lab, group headed by Dr. Alexander de Luna. Secondly at the Hubrecht Institute in Utrecht in the group of Prof. Alexander van Oudenaarden, and lastly at two groups at the University of Heidelberg, one at the Institute for Interdisciplinary Computing in the group of Prof. Fred Hamprecht, and a second one, currently working on my Master thesis at Bioquant in the group of Dr. Niels Grabe. 

BACHELOR: Firstly I explored gene-environment interactions using the yeast metabolism as a genetic-system model. Using the concept of epistatic genetic-interactions (i.e. the combined effect of two gene KO's is synergistic or antagonistic rather than additive) I explored the link between gene and environment for all metabolic single KOs in yeast (~ 480). To this end, I experimentally measured the impact in growth fitness phenotypes after the mentioned perturbations. I found that around 45% and 35%  of the essayed gene-knockouts had either a neutral or detrimental phenotype in comparison to the wild type in the same condition, while around 20%  have an enhanced fitness phenotype in at least one environmental perturbation. The latter observation unravels some of the evolutionary design principles of biological systems; the high number of knockouts which upon environmental remain neutral or acquire an advantage indicates a mechanism for exploring a genotype space. Consider, for instance a latent fraction of a population bearing a neutral mutation in X condition for which growth phenotype is enhanced by an abrupt environmental change to Y condition. The latent population fraction may be later positively selected under a detrimental environment for the native phenotype. While this oservations were not new, the exact distribution of the gene-environment interaction effects on fitness weren't previously systematically addressed. The development of this work took 11 months in the frame of my Bachelor thesis. My contributions to this project were varied, from experimental setup, to software develpment and data analysis. Experimentally, this project involved genetic manipulation of two collections of yeast knockout mutants which were generated (~900, ~480 with one replicate) by means of Synthetic Genetic Array methodologies. As well, the selection and validation of a consititive fluorescent marker was performed such that competition growth dinamics of mutant vs witld-type could be monitored in co-culture. Moreover, there was room for technology development, and a software platform was designed (in Python) to control a commercial robotic system (TECAN Freedom Evo 200); this allowed for the parallel automated measurments of fitness. Later, the quantitative and statistical analysis of the data was performed in MATLAB. All together, the achieved goal was the development of a platform to quantify the relative fitness of around 1300 competition assays in parallel. Finally we were able to carry out, collect and integrate data generated from more than 6000 competition assays.

Later, during the first National Synthetic Biology Summer Workshop which took place in the Center for Research and Advanced Studies of the National Polytechnic Institute in Mexico. Along with my research team, we were interested in the construction of a synthetic regulatory switch that supports memory by means of epigenetic modifications. I participated in the experimental design and in the use of Agent Based-modelling framewoirk to model the dynamics of the synthetic device. The system involved two competing site directed chromatin modifying factors with binfing sites at opposing sites of a set of fluorescent reporters. Although the summer did not suffice to put together the experimental handcraft, we did came up with an Agent Based Model to theoretically simulate the dynamics of our synthetic design.

After the completion of my bachelor I got temporary hired in a spin-off company founded by one of my professors at the university (Winter Genomics). In this just-started company my task was mainly to assess the usability of R packages in the tasks the company was focused on. During this short stay  (around 1 month) I reviewed and documented two packages, one for Genome Wide Association Studies and another, a dataframe schema for the representation of next-generation sequencing data. Despite my stay was very short, I used this time to refresh my knowledge of R and I also learned how to use very handy programs for collaborative projects and revision systems as is Git/Github.

MASTER: During my third semester of the master I 














There seems to be a connection
between plasticity (roughly, the environmentally induced change of phenotype
at constant genotype) and variability (roughly, the mutational change of phenotype at
constant environment). This has been shown to be the case for models of polymer structure
(such as RNA) [Ancel adn fontana], and predicts the possibility of a Baldwin effect at the molecular
level. A connection of this kind has also been hypothesized for more complex phenotypic
levels and processes, such as development. It may ultimately correspond to a general
insight from the theory of dynamical systems.

OVERALL SKILLS AND QUALIFICATIONS
During my senior year of undergraduate student I had the opportunity to join the Genetics Systems Lab; group headed by Dr. Alexander de Luna. There I started a research internship addressed to understand the plasticity of a phenotype under different environmental conditions. The project I developed allowed me to experience the pursuit of experimental research fully and in depth and this work also taught me the usage of a wide range of biochemistry and molecular biology techniques. In achieving this, I have shown myself to be self-motivated, committed and determined in successfully reaching my goals. I have also acquired group management skills as I directed the organization of a science workshop which required the correct coordination of voluntary personnel and the development of course content documentation. This last activity increased my sense of responsibility and my capacity to work hard under pressure. I possess excellent verbal and written communication skills, both in Spanish and English languages. I also have experience with German Language due to a high-school academic exchange for almost one year.

EXPERIMENTAL EXPERIENCE
As for my experimental qualifications, I have a good amount of wet lab experience. During the first 9-months of my one and a half year stay at Dr. de Luna's lab in the National Laboratory of Genomics for Biodiversity in Irapuato I learned how to perform molecular cloning of genes and almost all the habitual molecular-biology laboratory techniques. (PCR, gel electrophoresis, DNA and RNA extraction, etc.). My research project involved experimental and computational skills. I managed to analyze large measurement-datasets with tools like MATLAB and Python programming languages. 

Lately, as a part of my Master program I learned various quantitative methodologies, ranging from quantitative immunoblotting to mass spectrometry. Now, as a part of my lab rotation at the van Oudenaarden Lab, I am carrying out an experimental pipeline to quantify single molecule RNA FISH probes. This methodology starts by directly cultivating Mouse ES cells, all the way to image acquisition. Posteriorly, I analyse the images by a self coded MATLAB script.

THEORETICAL EXPERIENCE
Concerning research experiences in modeling, during the first National Synthetic Biology Summer Workshop which took place in the Center for Research and Advanced Studies of the National Polytechnic Institute in Mexico. Along with my research team, we were interested in the construction of a synthetic regulatory switch that supports memory by means of epigenetic modifications. I participated in the experimental design and in the use of ODEs to model the dynamics of the synthetic device. Although the summer did not suffice to put together the experimental handcraft, we did came up with both an agent based model and a simple implementation of the Gillespie algorithm in order to theoretically simulate the dynamics of our synthetic design.

All the Bachelor acquired modelling concepts were reinforced during the coursework in the first year of the Master program. There, I mostly learned the different paradigms in modelling using tools like COPASI and MATLAB. Modelling of both metabolic and signalling networks were the core of the course. Now, I am planning to join Prof. Niels Grabe from the University of Heidelberg to write my Master thesis and to learn how biological processes, as development, can be tackled by multi-scale approaches.

I am motivated by both, experimental and theoretical work, since what underlies my interest is motivated solely by the act of carrying out research. I would like to add, that my stay at different labs provided me with a teamwork spirit, and I believe that I have what it takes to forge personal relationships in order to work with other people and also to make positive contributions for the sake of a more comfortable work environment.


In my bachelor research project I used the yeast metabolism as a model to try to understand the interactions between genetic and environmental components underlying an advantageous gene knock-out (a knock-out that enhances fitness in certain condition). During the development of this project I became fascinated by the relationship of genotype-phenotype. This complex relationship, which is influenced by both, genetic interactions and gene-environment interactions, allowed me to dive into a new series of concepts like plasticity --how a biological system responds to environmental change-- and evolvability --the feasibility of a phenotypic change in response to mutation--. It has been observed that mutations often have consequences that vary across individuals. Moreover, current research indicates that environmental stimulation of a stress response can reduce mutation penetrance in C. elegans (Casanueva et al., 2012) , Casanueva et al. experimentally observed that the induced mutation buffering varies across genetically identical individuals because of individual differences in stress signaling, indicating that this flexibility in signal transduction can induce protection against mutations, which might be facilitating further adaptation (Espinosa-Soto et al., 
2 2011). 

Now I am doing a lab rotation at the van Oudenaarden Lab, at the Hubrecht Institute in Utrecht, NL. These last months I've been mostly into image processing: tissue segmentation and feature quantification; in the lab I am working now, they have developed a very quantitative methodology to count single mRNA molecules using microscopy. They are using this technique to assess the expression of genes tightly regulating development, or the variability of these in a single cell basis. 

Lately I've discovered that the development of multicellular organisms is a truly plastic process and is also well suited for a systems biology approach (as the robust formation of magnificent patterns has always wondered scientists, I am no exception). I think that it is the modelling of the cell's inner circuitry in combination with topological cues (structure, population context, etc.) what makes of the multi-scale modelling paradigm a very suitable framework to study the plasticity development. I would like to learn a handful of tools addressed to this kind of modelling, it doesn't have to be directly linked to development in this case. 

All this concepts together, constitute the basic context of my current interest. Now, as for the MDC program i am interested in the group of Prof. Loewer, mostly because I think that the relationship between genotype and phenotype can only be explained by studying the combinatorics of signal transduction from a single cell perspective. 


[1]Fitness Trade-Offs and Environmentally Induced Mutation Buffering in Isogenic C. elegans M. Olivia Casanueva, Alejandro Burga, and Ben Lehner. Science, 6 January 2012:335 (6064), 82-85 
[2]Phenotypic plasticity can facilitate adaptive evolution in gene regulatory circuits Carlos Espinosa-Soto, Olivier Mertin and Andreas Wagner. BMC Evolutionary Biology, 6 January 2011, 11:5



## Scientific Interests

Phenotypic features constraining/directing evolutionary trajectories of genes. Evolutionary trajectories of genes depend on their mutations and the distribution of the associated fitness effect.in order to find genotypes with improved phenotypes, or phenotype that match the required environmental demands, the genetic capacitors can be seen as the genes responsible for the canalization of a phenotype, so that they are the key element in a genotype network. ando how a canalization property is mapped into the fitness landscape. does this canals or atractors in the phenotype map, respectively map to peaks in the fitness landcape? As environmental capacitors, we have observed that chaperones play an important role, and also very important that they are buffering genetic perturbations, We observe one interaction between two genes, this interaction can either be neutral, aggravating or beneficial. We would also expect the appearance of one expression phenotype combined with one loss-of-function phenotype and expect the same three resulting scenarios? We say we might be helping the organism to optimize its fitness by previously deleting one gene, then exposing it to the environmental change will increase its chance to survive relative to the native genotype. The ontogenic process leading to differential mutation penetrance, or differential phenotype MM Canalization is the reduction in sensitivity of a phenotype to perturbation, either genetic or environmental. MM Phenocopy is a variant that is produced in response to environmental perturbation and that mimics a variant phenotype that can be produced by mutation
In my bachelor research project I used the yeast metabolism as a model to try to understand the interactions between genetic and environmental components underlying an advantageous gene knock-out (a knock-out that enhances fitness in certain condition). During the development of this project I became fascinated by the relationship of genotype-phenotype. This complex relationship, which is influenced by both, genetic interactions and gene-environment interactions, allowed me to dive into a new series of concepts like plasticity –how a biological system responds to environmental change– and evolvability –the feasibility of a phenotypic change in response to mutation–. It has been observed that mutations often have consequences that vary across individuals. Moreover, current research indicates that environmental stimulation of a stress response can reduce mutation penetrance in Caenorhabditis elegans (Casanueva et al., 2012)Fitness Trade-Offs and Environmentally Induced Mutation Buffering in Isogenic C. elegans M. Olivia Casanueva, Alejandro Burga, and Ben Lehner. Science, 6 January 2012: 335 (6064), 82-85, Casanueva et al. experimentally observed that the induced mutation buffering varies across genetically identical individuals because of individual differences in stress signaling, indicating that this flexibility in signal transduction can induce protection against mutations, which might be facilitating further adaptation (Espinosa-Soto et al., 2011)Phenotypic plasticity can facilitate adaptive evolution in gene regulatory circuits Carlos Espinosa-Soto, Olivier Mertin and Andreas Wagner. BMC Evolutionary Biology, 6 January 2011, 11:5 . 

All this concepts together, constitute the basic context of my current interest. Now, during the Master program in Systems Biology in the University of Heidelberg I aim to study whether the combinatorial effect within a signaling pathway plays a role mechanistically explaining how environmental responses can underlie variable mutation buffering, and how this plasticity makes variation in stress signaling advantageous. To do this, recently developed frameworks like the rule based modeling-language Kappa, offers a suitable approach. 


