Title:Introduction
Summary: The introduction to my report
Category: projects
Tags: biology, research, hd dna-fish, master, report
Date: 2013-03-28
Abstract: The control of gene expression in eukaryotes can be viewed as the integrated response of mechanisms working at different hierarchical levels. Gene transcription is regulated from the sequence level to chromatin level ...

The control of gene expression in eukaryotes can be viewed as the integrated response of mechanisms working at different hierarchical levels. Gene transcription is regulated from the sequence level to chromatin level, to the nuclear level (van Driel et al. 2003). At this top-most layer, the higher order topology of the genome in the nucleus has proven to play an important role, as meaningful patterns of organization have been identified; gene-poor chromosomes are generally more frequently positioned in the outer regions of the nucleus, whereas gene-rich chromosomes adopt a more internal localization (Croft et al. 1999). Moreover, motion of genes along the radial axis of the nucleus have been observed upon gene activation, either toward the nuclear interior or toward the nuclear periphery being the former not exclusively a repressive location (Andrulis et al. 1998, Boyle et al. 2001, Casolari et al. 2005, Mesiter et al. 2010). Alongside, distributed throughout the nucleus, sets of activated genes are spatially arranged into discrete foci called transcription factories dedicated to the expression of specific combinations of genes even from distinct chromosomes (Spilianakis et al. 2005, Osborne et al. 2007, Schoenfelder et al. 2009 and 2010). This indicates that the associations at the transcriptional hot spots does not happen at random, but genes have preferential partners, commonly in a tissue specific manner. In addition, significant changes in spatial organization of genomes are triggered during cellular differentiation reflecting a dramatic change in gene expression (Kim et al. 2004).

Currently the chromosome is viewed as an organelle consisting of non-random, differentially and tissue specific positioned arrangement of chromosomes into functionally distinct sub-domains called chromosome territories (Cremer et al. 2006). The spatial context of a gene within the nucleus as well as within a chromosome appears to have an important role on transcriptional control.Various cell-types may possess different chromatin arrangements accounting for variability in multicellular organisms, The impact of higher order nuclear architecture on these patterns in yet unknown.()

chromatin organization requires a comprehensive measurement of the landscape of interacting genomic regions.

>transcription plays a major role in shaping the nuclear organization of the genome (plagio)

###Examples:

* Proper silencing of genes involved in B cell and T cell development is dependent upon the ability of the genes to relocate away from the periphery of the nucleus. (Kosak et al. 2002).

* Radial motion of genes upon activation: 
	
	* proper silencing of the mating type in _S. cerevisiae_ (HMR and HML loci is dependent upon their ability to associate with the nuclear periphery (Andrulis et al. 1998).

	* transcriptional activation of the GAL genes result in their association with nuclear pore proteins and relocation to nuclear periphery (Casolari et al. 2004). 
	
	* \\alpha-factor-induced genes become associated with components of the nuclear envelope (Casolari et al. 2005)

	* Tissue-specific genes on differentiated cells in _C. elegans_ shift toward the nuclear lumen when activated or to the nuclear envelope when silent (Mesiter et al. 2010)

	* In addition, human lymphocytes show a strong correlation between the radial position of human chromosomes and their gene density, with gene-poor chromosomes positioned toward the nuclear periphery and gene-rich chromosomes located in the nuclear interior (Boyle et al., 2001)

* Spatial arrangement of activated gene into transciption factories, these factories or subcompartments are dedicated to even the transcription of  specific combinations of genes even from distinct chromosomes. Suggesting that active genes might have preferred transcription partners.
	
	* Non-nucleolar active polymerases are confined to discrete foci termed transcription factories which are distributed throughout the nucleus. However the specific mechanisms by which genes are recruited to these sites is yet not understood.(Schoenfelder et al. 2010)

	* Studies of the transcriptional dynamics of the immediate early genes Fos and Myc showed that signaling-induced expression of these genes involves very rapid nuclear relocation to factories; Myc proto-oncogene on Chromosome 15 is preferentially recruited to the same transcription factory as the highly transcribed Igh gene located on Chromosome 12 upon gene induction. genes migrate to preassebled transcription sites (Osborne et al. 2007).

	* Klf1-regulated genes cooperate to create nuclear transcriptional hot spots (Schoenfelder et al. 2009)

	* >_It should be noted that most ‘active’ genes are not continuously producing transcripts. Visualization of mammalian gene transcription by RNA FISH suggests it is a binary process that occurs in infrequent bursts. Gene activation is associated with relocation of genes to transcription factories, indicating that transcriptional pulses occur by virtue of dynamic associations with transcription factories._ __(plagio, Schoenfelder et al. 2010,  see also Osborne et al. 2007)__


>**can we derive a comprehensive model of how genomes are organized and function in vivo?** 

>_Such a model should account for the complex morphological features of the nucleus and should be consistent with the structural and dynamic properties of genomes_

>The central idea is that the sum of all functional properties of a chromosome (i.e., the frequency and linear distribution of its active and inactive regions) determines its positioning.The sum of these interactions creates preferential associations amongst genome regions and chromosomes and constrains their motion. In this way, each genome region and each chromosome determines in a self-organizing fashion whom its neighbor is, and preferential, yet probabilistic, patterns of positioning emerge __(Misteli 2007)__

Currently two frameworks have been widely used to study chromatin organization in the context of the nucleus. 

>A major question in understanding spatial genome organization is how patterns of chromosome or loci organization are established and maintained. Two extreme models can be envisioned. It can be argued that the transcriptional activity of a genome region determines the physical properties of the chromatin. It is well established that transcriptionally active chromatin is generally more decondensed than inactive chromatin. The physical properties of chromatin and the sum of interacting proteins, such as polymerases, might determine the distribution of chromosomes and genes in a self-organizing manner (Cook, 2002). For example, it is possible that the decondensed, active regions of chromatin physically push the inactive heterochromatin regions towards the periphery. Furthermore, the interaction of active regions with polymerases might provide anchor points, which determine the spatial organization of the genome (Cook, 2002). In contrast to such a self-organization model is the possibility of defined, molecular organizers, which might anchor particular genome regions to specific places within the nuclear architecture. (Kim et al. 2004)

checkout:The role of transcription factories in large-scale structure and dynamics of interphase chromatin