## Chromatin topology plays an important role in gene regulation
![slide1](/static/images/slides/chromatin-organization/slide1_2.png)
## Distance between chromatin regions are measured from 3D positions of fluorescent spots
![slide2](/static/images/slides/chromatin-organization/slide2.png)
## Spots are identified for singles cells
![spotterUI2](/static/images/slides/chromatin-organization/spotterUI.png)
## Precise localization of the spots is estimated and corrected for chromatic aberrations
![gaussfit](/static/images/slides/chromatin-organization/slide4.png)
## Distances within single chromosomes are measured
![distances](/static/images/slides/chromatin-organization/slide5.png)
## Our results are in agreement with previous observations
![4 loci counts][4_loci_level]

<pre style="text-align:right;">Mateos-Langerak et al. 2009 PNAS</pre>

[4_loci_level]:/static/images/slides/chromatin-organization/slide6.png
### Outlook
#### Combinatorial labeling provides identifiers for unique regions
![distances](/static/images/slides/chromatin-organization/comb-label.png)
## Concluding remarks

* A major question in understanding spatial genome organization is how patterns of chromosome or loci organization are established and maintained

* Better resolution and perhaps multiplexing for the precise estimation of the localization of chromosome regions can be acheived by using our methodology 

* These measurements will provide insight in understanding the formation of patterns in chromatin organization

## Aknowledgements

### Group van Oudenaarden

#### at MIT:
* Magda Bienko
* Stefan Semrau

#### at the Hubrecht Institute:
* Alexander van Oudenaarden

## try come code:

	::latex::
	begin{equation}
	   E = mc^2
	   e^{\pi i}+1 = 0 \frac{x}{y}
	end{equation}
### is translated into:

\begin{equation}
E = mc^2 \\
e^{\pi i}+1 = 0 \frac{x}{y}
\end{equation}

</div><div class="slide" id="30" style="background-color:#000000;"><h1 style="color:#fafafa;">Thank you!</h1>